packages/dds/tree/src/simple-tree/api/schemaFactoryAlpha.ts

```diff
@@ -188,9 +193,16 @@ export interface SchemaStaticsAlpha {
 	 *
 	 * TODO: AB#45711: Update the docs above when recursive type support is added.
 	 */
-	staged: <const T extends LazyItem<TreeNodeSchema>>(
+	readonly staged: <const T extends LazyItem<TreeNodeSchema>>(
 		t: T | AnnotatedAllowedType<T>,
 	) => AnnotatedAllowedType<T>;
+
+	readonly types: <
```

Comment:
This is the new API for bundling annotated types into the new (and backwards compatible) AllowedTypesFull.

Reply:
I would prefer to avoid it, but doing so would add complexity elsewhere, which I don't think is worth doing at this time. At some point, once we have staged stabilized, and recursive staged types working, we could revisit this and see if adding support for annotated allowed type in more places to allow skipping this is practical.